serverless/serverless-release-notes.adoc

```diff
@@ -22,6 +27,14 @@ include::modules/serverless-tech-preview-features.adoc[leveloffset=+1]
 include::modules/serverless-deprecated-removed-features.adoc[leveloffset=+1]
 
 // Release notes included, most to least recent
+// OCP + OSD + ROSA
+include::modules/serverless-rn-1-26-0.adoc[leveloffset=+1]
+// 1.26.0 additional resources
+[role="_additional-resources"]
+.Additional resources
+* link:https://knative.dev/docs/eventing/experimental-features/new-trigger-filters/
```

Comment:
Looks like there's a gap between the `link:` here and the text on the next link, making it render verbatim.